docs/jsonrpc_apis.md

```diff
@@ -579,20 +570,19 @@ argument sample:
 
 result sample:
 
-```javascript
+```json
 {
-  "id": null,
-  "error": null,
-  "jsonrpc": "2.0",
-  "result": [
-    {
-      "Time": 1524798750979702000,
-      "Services": 4,
-      "IP": [0,0,0,0,0,0,0,0,0,0,255,255,127,0,0,1],
-      "Port": 30338,
-      "ID": 8775829619427993046
-    }
-  ]
+    "error": null,
+    "id": 123456,
+    "jsonrpc": "2.0",
+    "result": [
+        "127.0.0.1:64890 (inbound)",
+        "127.0.0.1:64880 (inbound)",
+        "127.0.0.1:64822 (inbound)",
+        "127.0.0.1:22338 (outbound)",
+        "127.0.0.1:23338 (outbound)",
+        "127.0.0.1:24338 (outbound)"
+    ]
```

Comment:
The result is not matched to the function.

Reply:
The result has changed after merge dpos into dev, so the document needs to update.
![image](https://user-images.githubusercontent.com/11372825/53631372-59e77480-3c4d-11e9-9a6a-753932f5b9a1.png)

Reply:
Please add a description of the result.

Reply:
The result is just an address in string, no like the GetNodeState() interface the result is struct and have many  parameters.

Reply:
Maybe you can refer to this document [getneighbors](https://github.com/elastos/Elastos.ELA.SideChain.Token/blob/release_v0.0.1/jsonrpc_apis.md#getneighbors)